sleepyq/__init__.py

```diff
@@ -80,17 +80,20 @@ def __init__(self, login, password):
         self._login = login
         self._password = password
         self._session = requests.Session()
-        self._api = "https://api.sleepiq.sleepnumber.com/rest"
+        self._api = "https://prod-api.sleepiq.sleepnumber.com/rest"
```

Comment:
Does this mean the existing library is broken now? 🤔

Reply:
The old URL still works, but the web interface has changed it's URL. I updated it as a preventive measure.